src/Clinic.DAL/Entities/EnumsEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Clinic.DAL.Entities
+{
+    public class EnumsEntity
+    {
+        [JsonConverter(typeof(JsonStringEnumConverter))]
+        public enum OfficeStatus
+        {
```

Comment:
Add deafult value for 0
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/enum